### PR TITLE
kernel/binary_manager: Change the structure name of binary update information

### PR DESCRIPTION
**Pull request by jeongarmy**
Change the structure name of binary update information

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [c33047086c17d4fa04acd39d0338415e94b99fd2] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [c33047086c17d4fa04acd39d0338415e94b99fd2] - Code Rule Check OK.

File: os/include/tinyara/binary_manager.h [hunk 1]
```diff
@@ -107,15 +107,15 @@ enum binmgr_response_result_type {
 /****************************************************************************
  * Public Data
  ****************************************************************************/
-/* The structure of binary information */
-struct binary_info_s {
+/* The structure of binary update information */
+struct binary_update_info_s {
 	int part_size;
 	char name[BIN_NAME_MAX];
 	char version[BIN_VERSION_MAX];
```
Comment (sunghan-chang) [inline os/include/tinyara/binary_manager.h]: As we discussed, you should add prefix of active or inactive

**Comment by seinfra** [comment]
Target : [7335261ae3fef134b5d12351bac58371bd082eb9] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [7335261ae3fef134b5d12351bac58371bd082eb9] - Code Rule Check OK.

**Comment by jeongarmy** [comment]
@btheosam I know what you mean. This data is not information about update and is needed for binary update.
I named it because binary_update_info_t is used in binary_update(prefix)_get_info which is a function to get information needed for binary update.

**Pull request merged**